site2/docs/functions-state.md

```diff
@@ -9,9 +9,11 @@ for storing the `State` for functions. For example, A `WordCount` function can s
 
 ## API
 
+<!--DOCUSAURUS_CODE_TABS-->
```

Comment:
Thank you very much for adding those valuable info. Could we add those info in the [functions-develop.md#state-storage] section?